mk64.ld

```diff
@@ -454,7 +454,14 @@ SECTIONS
    /* compressed kart textures and palettes */
    BEGIN_SEG(kart_textures, 0x0F000000)
    {
-      BUILD_DIR/data/kart_textures.o(.data);
+      BUILD_DIR/data/karts/luigi_kart.o(.data);
+      BUILD_DIR/data/karts/mario_kart.o(.data);
+      BUILD_DIR/data/karts/yoshi_kart.o(.data);
+      BUILD_DIR/data/karts/peach_kart.o(.data);
+      BUILD_DIR/data/karts/wario_kart.o(.data);
+      BUILD_DIR/data/karts/toad_kart.o(.data);
+      BUILD_DIR/data/karts/donkeykong_kart.o(.data);
+      BUILD_DIR/data/karts/bowser_kart.o(.data);
       BUILD_DIR/courses/staff_ghosts.inc.o(.data);
```

Comment:
Do you think it might make sense to move staff_ghosts out of courses? (maybe not for this PR just wondering what you think)

Or does it still seem sensical for it to stay?

I have it this way cause the individual staff ghost file is in its relevant course folder.

Reply:
I honestly don't know. I'd have to look at the file to see what that stuff is really doing. If its the code that handles "playing" the ghost in time trial mode it might make sense to move it into `src`. But I think you've touched that stuff the most so you're the one who'd know best.

Reply:
It's just the controller data for the staff ghost. Like A button, left, right, etc. The actual code is in src/staff_ghosts.c